### DISCUSSION TITLE
How to not trigger aggregation logic given conditions

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#129] [Q&A] [answered]
Hi,
If it's possible, what is the recommended way to filter the ingestion messages in a way that they don't trigger the aggregation logic?

There are projections that given one or multiple conditions, I don't want the related single view to be re-generated.

**Comment by frengooo** [comment, marked as answer]
Hi @elmalakomar,
I don't know if you can fix in this way but can't you just delete the autogenerated strategy through this iter?
- Single Views page
- involved single view
- strategies page

Like the following screenshot:
<img width="1708" alt="Screenshot 2022-12-01 alle 10 21 55" src="https://user-images.githubusercontent.com/28152586/205015466-f8955d42-6ffd-46d0-8c9b-25f0211ac1b3.png">

I don't know if it's enough for you, otherwise I think you need to use custom functions inside the `projectionChangesSchema.json` file with the `__fromFile[filename]__` syntax reported [on the documentation](https://docs.mia-platform.eu/docs/fast_data/configuration/realtime_updater/low_code#custom-path-of-strategies).

Hope this helps you.
